README.rst

```diff
@@ -23,15 +23,15 @@ Automatically upgrade your Django project code.
 Installation
 ============
 
-Use **pip**:
+**Option 1**: Install django-upgrade with **pip**:
```

Comment:
maybe we could use real headings rather than “option 1” / ”option 2”?

Reply:
Sure, I couldn't figure out what to write between the "Installation" heading and "Option 1...". I prefer at paragraph between to headings. Will add some simple one-liner.